examples/bvh_driver/CMakeLists.txt

```diff
@@ -23,4 +23,4 @@ find_package (Threads REQUIRED)
 add_executable(ArborX_BoundingVolumeHierarchy.exe bvh_driver.cpp)
 target_link_libraries(ArborX_BoundingVolumeHierarchy.exe ${ArborX_TARGET} benchmark::benchmark Boost::program_options Threads::Threads)
 target_include_directories(ArborX_BoundingVolumeHierarchy.exe PRIVATE ${POINT_CLOUDS_INCLUDE_DIR} ${UNIT_TESTS_INCLUDE_DIR})
-add_test(NAME ArborX_BoundingVolumeHierarchy_Example COMMAND ./ArborX_BoundingVolumeHierarchy.exe)
+add_test(NAME ArborX_BoundingVolumeHierarchy_Example COMMAND ./ArborX_BoundingVolumeHierarchy.exe --buffer=0 --benchmark_color=true)
```

Comment:
Does the coloring works with Jenkins/ctest or do we get something unreadable where we get a bunch of color tags in the log file?

Reply:
I haven't checked.  I picked randomly one of the options to make sure it works.

Reply:
Do you want me to change to `auto`?

Reply:
I don't care what we use as long as the log is readable

Reply:
> I don't care what we use as long as the log is readable

Yeah...  About that.  Have you looked at the logs on Jenkins?  This is another of these need to fix

Reply:
The log file is deleted anyway because it is too large...